ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -16,8 +16,8 @@ def sign(x, /):
 
 
 @to_ivy_arrays_and_back
-def absolute(x, /):
-    return ivy.abs(x)
+def absolute(x, /, where):
+    return ivy.abs(x, where=where)
```

Comment:
We shouldn't be changing the frontends in this PR, `jax.numpy.absolute` doesn't accept a `where` argument anyway